### ISSUE TITLE
Can not map SQL type 3 (DECIMAL) to ClickHouse

### ISSUE THREAD

**Issue by ramazanpolat** [ClickHouse/clickhouse-jdbc-bridge#14]
Trying to use it to connect to IBM DB2.

Connection is successful but if the remote table has a column type of `DECIMAL` then we get `Can not map SQL type 3 (DECIMAL) to ClickHouse.`.

Tried several cases, in all cases, if the table contains a column that is typed ad `DECIMAL`, this error pops up.

**WORKAROUND**:

Create a view that converts `DECIMAL` values to `BIGINT`.

**Comment by long2ice** [comment]
same here

**Comment by zhicwu** [comment]
I believe this has been fixed in 2.0,  which will be released in Dec.

**Comment by zhicwu** [comment]
Please try release 2.0.0 as below works for me:
```sql
select * from jdbc('postgresql?mutation', 'drop table if exists test_table')
select * from jdbc('postgresql?mutation', 'create table if not exists test_table(d decimal(10,3))')
select * from jdbc('postgresql?mutation', 'insert into test_table values(1.23)')
select * from jdbc('postgresql', 'd Decimal(10,3)', 'select * from test_table')
```
